registry.json

```diff
@@ -1,5 +1,18 @@
 [
-
+   {
+       "ticker": "SHARK",                                                       
+       "poolId": "672f8a27fe8c5115ba67a68d04c81c8604cb68e3d0590d7c085f5a28",  
+       "social": {              
+         "website":"https://medium.com/@metash4rk", 
+         "twitter": "@Meta__Shark",
+         "facebook": " ",  
+         "instagram": " ",
+         "youtube": " "
+       },
+       "telegram": "metash4rk",                                             
+       "operatorName": "sawhney",  
+       "github": "https://github.com/meta-shark"
```

Comment:
Please can your remove the host from your github? You can just leave this as "meta-shark"

Reply:
done !